### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by drupol — APPROVED** [review]
Documentation is badly needed, thanks mate.

**Comment by ghost** [comment]
It's just the basics but it's a start. I'll look at the other ones tomorrow.

**Comment by drupol** [comment]
Feel free to commit/merge as you're maintainer. Documentation doesn't need my approval.

**Pull request merged**
